kvbc/test/sparse_merkle/tree_test.cpp

```diff
@@ -274,7 +274,7 @@ TEST(tree_tests, insert_single_leaf_node) {
   // Only a single node was created. It's key is made up of an empty nibble
   // path, since there are no collisions requring going deeper into the tree.
   auto& root = batch.internal_nodes.at(0);
-  ASSERT_EQ(InternalNodeKey(1, NibblePath()), root.first);
+  ASSERT_EQ(InternalNodeKey("", 1, NibblePath()), root.first);
```

Comment:
do you have tests that create multiple trees?

Reply:
yes, a test covering multiple MPT-s is implemented here [PR-3007](https://github.com/vmware/concord-bft/pull/3007)